lib/did_you_mean/levenshtein.rb

```diff
@@ -0,0 +1,57 @@
+module DidYouMean
+  module Levenshtein # :nodoc:
```

Comment:
What about to move levenshtein to lib/levenshtien.rb and make it proper standard library item?

Reply:
Sounds like something we can iterate on. Would appreciate if you could open an issue on https://bugs.ruby-lang.org/projects/ruby-trunk/issues.

Reply:
yup, I'll open issue in there covering this idea